internal/export/exportfile.go

```diff
@@ -84,8 +86,51 @@ func MarshalExportFile(eb *model.ExportBatch, exposures []*publishmodel.Exposure
 	return buf.Bytes(), nil
 }
 
+// Unmarshals export file and extract exposure keys information.
+func UnmarshalExportFile(payload []byte) (*export.TemporaryExposureKeyExport, error) {
+	zp, err := zip.NewReader(bytes.NewReader(payload), int64(len(payload)))
+	if err != nil {
+		return nil, fmt.Errorf("Can't read payload: %v", err)
+	}
+
+	for _, file := range zp.File {
+		if file.Name == exportBinaryName {
+			return unmarshalContent(file)
+		}
+	}
+
+	return nil, fmt.Errorf("Payload is invalid: no %v file was found", exportBinaryName)
+}
+
+func unmarshalContent(file *zip.File) (*export.TemporaryExposureKeyExport, error) {
+	f, err := file.Open()
+	defer f.Close()
+	if err != nil {
+		return nil, err
+	}
+
+	content, err := ioutil.ReadAll(f)
+	if err != nil {
+		return nil, err
+	}
+
+	prefix := content[:fixedHeaderWidth]
+	sprefix := string(prefix)
```

Comment:
If we make fixedHeader as []byte, we can do if !bytes.Equal and will avoid the many bytes->string conversions.

Reply:
We can't declare a constant with []byte. 
Should we use a variable for this?

Reply:
I'd go for it, reduces conversion costs but moreover is usable only in one place in code, aside from tests.

Reply:
Actually usable in 2 places and prevents unnecessary conversions in 2 places :)